site/users/index.html.js

```diff
@@ -48,6 +48,16 @@ var logos = [
     img: 'reindex.png',
     link: 'https://www.reindex.io/'
   },
+  {
+    name: 'Meteor',
+    img: 'meteor.png',
+    link: 'https://medium.com/apollo-stack/snappier-uis-with-apollo-client-graphql-bffab0b52b79#.sls7hdvud'
```

Comment:
This looks like it also links to Apollo.

Maybe it would be better to just pick one of these two logos? This page should be a list of companies, not a list of products, so the Meteor logo is a better fit. We've got references to the Apollo project elsewhere in the docs :)